### PR TITLE
Create runbooks for new OnCall alerts

### PR DESCRIPTION
**Pull request by TheBigLee**
## Summary

This PR adds runbooks for the new onCall alert for our SPKS offering

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[WARNING]
+====
+Make sure you boostrap from the *last* healthy pod. There are multiple options to find the last healthy pod:
+
+- Create a debug mariadb to mount the PVC and follow these https://mariadb.com/kb/en/galera-cluster-recovery/[instructions].
+
+====
+
+IMPORTANT: If the cluster is down, due to full PVC storage, you first have to resize the underlying PVC.
+This needs to be done by the customer, by choosing a higher instance tier.
```
Comment (bastjan) [inline docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc:21]: External dependencies (stuff we can't fix ourselves) should never escalate on-call.

Comment (TheBigLee) [reply docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc:21]: Well it's almost impossible to automatically figure out based on metrics, what the exact issue is when an instance goes down.

Comment (Kidswiss) [reply docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc:21]: I agree, getting woken up by a non-actionable alert is not coll.

File: docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+== icon:bug[] Steps for Debugging
+
+See xref:how-tos/mariadbgalera/debug.adoc[How to debug MariaDB Galera] on how to determine the cluster status and specifically xref:how-tos/mariadbgalera/bootstrap_cluster.html[Boostrap a cluster] on how to bring the cluster back up.
+
+[WARNING]
+====
+Make sure you boostrap from the *last* healthy pod. There are multiple options to find the last healthy pod:
+
+- Create a debug mariadb to mount the PVC and follow these https://mariadb.com/kb/en/galera-cluster-recovery/[instructions].
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc:16]: Would be nice if we'd have a ready to copy paste snippet here that prints out the safe node. You don't want to read upstream docs at 3 in the morning.

File: docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc (hunk 1 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc]: I removed that section from the docs again and adjusted our alert a bit to not trigger in this case.

File: docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc (hunk 2 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterDown.adoc]: I added a copy&pasteble Pod definition to the docs that should spit out the correct node to bootstrap from.

**Review by Kidswiss — APPROVED** [review]

**Pull request merged**